build.sbt

```diff
@@ -1,24 +1,16 @@
-import sbt.Keys._
-import sbt._
 import uk.gov.hmrc.DefaultBuildSettings._
-import uk.gov.hmrc.versioning.SbtGitVersioning
-import uk.gov.hmrc.SbtArtifactory
 
-val appName: String = "release-versioning"
-
-lazy val releaseVersioning = Project(appName, file("."))
-  .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning, SbtArtifactory)
+lazy val releaseVersioning = Project("release-versioning", file("."))
   .settings(scalaSettings: _*)
   .settings(defaultSettings(): _*)
   .settings(
-    scalaVersion := "2.11.12",
+    scalaVersion := "2.12.14",
     crossScalaVersions := Vector("2.10.7", "2.11.12", "2.12.8"),
     majorVersion := 0,
-    makePublicallyAvailableOnBintray := true,
+    isPublicArtefact := true,
     libraryDependencies ++= compileDependencies ++ testDependencies,
     retrieveManaged := true,
```

Comment:
Do we need this?

Reply:
Probably not. Looks like it was broken with coursier anyway (https://github.com/sbt/sbt/issues/5078)